Moonscraper Chart Editor/Assets/Scripts/Game/GameSettings.cs

```diff
@@ -298,6 +300,7 @@ public GameSettings()
     {
         slowdownPitchCorrectionEnabled = new BoolSaveSetting(SECTION_NAME_SETTINGS, "Slowdown Pitch Correction Enabled", false);
         extendedSustainsEnabled = new BoolSaveSetting(SECTION_NAME_SETTINGS, "Extended sustains", false);
+        openTapChordsEnabled = new BoolSaveSetting(SECTION_NAME_SETTINGS, "Open Chords & Tap Opens", false);
```

Comment:
Would probably be better to split open chords and tap opens into two different feature sets, at least on the backend, even if we present this as one option to the user.